lib/Api/v2/InventoryUnitApi.php

```diff
@@ -917,11 +917,11 @@ public function getInventoryUnits($limit = '100', $page = '1')
      *
      * @throws \SMS\Suppliers\ApiException on non-2xx response
      * @throws \InvalidArgumentException
-     * @return array of object, HTTP status code, HTTP response headers (array of strings)
+     * @return array of \SMS\Suppliers\Model\v2\GetInventoryUnitResponse[], HTTP status code, HTTP response headers (array of strings)
      */
     public function getInventoryUnitsWithHttpInfo($limit = '100', $page = '1')
     {
-        $returnType = 'object';
+        $returnType = '\SMS\Suppliers\Model\v2\GetInventoryUnitResponse[]';
```

Comment:
Can we switch to `::class` where we reference a class name?

```suggestion
        $returnType = GetInventoryUnitResponse::class.'[]';
```

Reply:
Since the code is generated by `make generate-suppliers-api-sdk`, I would like to keep it as it is. See also [the documentation how to generate the SDK](https://hitmeister.atlassian.net/wiki/spaces/SMS/pages/488112671/API#How-to-generate-API-SDK).